tests/test_printer.py

```diff
@@ -222,6 +222,11 @@ def test_piecewise_expressions(self, p, x):
             evaluate=False)
         assert p.doprint(e) == '((0) if (x > 0) else (2))'
 
+        # First condition false, multiple true clauses
+        e = sp.Piecewise((6, False), (0, x < 5), (1, True), (2, True))
+        print(e)
+        assert p.doprint(e) == '((0) if (x < 5) else (1))'
+
```

Comment:
So you also want to test
```
        # Middle condition only true
        e = sp.Piecewise((0, x < 5), (1, True), (2, x > 7))
        assert p.doprint(e) == '((0) if (x < 5) else (1))'
```

Reply:
Yes that works fine!

It might seem to change the logic but that's not actually the case @jonc125 The reason is that sympy seems to filter out clauses after the True clause by the time they get to the printer so if I call
```
# Middle condition only true
e = sp.Piecewise((0, x < 5), (1, True), (2, x > 7))
p.doprint(e)'
```
then inside:
`_print_Piecewise` if I get  `expr`:
`Piecewise((0, x < 5), (1, True))`

Having said all that I don't really understand why adding the break makes it show up as a miss (or BrPart) in the coverage

Reply:
It'll be because the `break` is never *not* executed, so only one branch in the logic is taken. Similar to an `if` always being true, which is a coverage issue even with no `else` clause.

Reply:
Aah I see ok let me fix that